panel/widgets/slider.py

```diff
@@ -121,32 +124,73 @@ def _get_embed_state(self, root, values=None, max_opts=3):
         return (dw, w_model, values, lambda x: x.value, 'value', 'cb_obj.value')
 
 
-class FloatSlider(ContinuousSlider):
+class FloatSlider(_ContinuousSlider):
+    """The FloatSlider widget allows selecting a floating-point value within a
+    set of bounds using a slider.
 
-    start = param.Number(default=0.0)
+    Reference: https://panel.holoviz.org/reference/widgets/FloatSlider.html
 
-    end = param.Number(default=1.0)
+    ## Example
 
-    value = param.Number(default=0.0)
+    ```python
+    import panel as pn
 
-    value_throttled = param.Number(default=None, constant=True)
+    pn.extension()
 
-    step = param.Number(default=0.1)
+    slider=pn.widgets.FloatSlider(value=0.5,start=0.0,end=1.0,step=0.1, name="Float Value")
```

Comment:
I'd rather format this with spaces and without assigning to a variable.
```
pn.widgets.FloatSlider(value=0.5, start=0.0, end=1.0, step=0.1, name="Float Value")
```

Reply:
I did assign to slider (and include `import panel` + `pn.extension` because then it was easy to copy code to use. I've removed it.

The space is there because that is how Black formats python code. I would argue we should keep the space 😄

Reply:
I've removed the `slider =`

Reply:
@MarcSkovMadsen unless I'm wrong in your code there were no spaces around the equal sign and after most commas, I'd be very surprised to see black format it this way since I assume it's still very close to PEP8. What black may do is to break the code into multiple lines, not sure that would be a good idea here.

Reply:
I don't know what black you are running @MarcSkovMadsen but my black gives the same output as @maximlt with spaces after comma and not after equal.